wiremock/stubApis.ts

```diff
@@ -39,10 +40,29 @@ premises.forEach(p => {
   )
 })
 
+stubs.push(async () =>
+  stubFor({
+    request: {
+      method: 'POST',
+      urlPathPattern:
+        '/premises/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/bookings/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/arrivals',
+    },
+    response: {
+      status: 200,
```

Comment:
should this be a `201`?

Reply:
Whoops - yeah! I think that was down to a bad merge.